consensus/core/src/core.rs

```diff
@@ -275,11 +272,34 @@ impl Core {
     /// Adds/processed all the newly `accepted_blocks`. We basically try to move the threshold clock and add them to the
     /// pending ancestors list.
     fn add_accepted_blocks(&mut self, accepted_blocks: Vec<VerifiedBlock>) {
-        // Advance the threshold clock. If advanced to a new round then send a signal that a new quorum has been received.
-        if let Some(new_round) = self
-            .threshold_clock
-            .add_blocks(accepted_blocks.iter().map(|b| b.reference()).collect())
+        // Get max round of accepted blocks. This will be equal to the threshold
+        // clock round, either by advancing the threshold clock round by being
+        // greater than current clock round or by equaling the current clock round.
```

Comment:
Is this the case? Blocks older than current threshold clock round can get accepted as well.

Reply:
I only added the case in the comment for greater and equal but blocks less than the clock round are essentially ignored by threshold clock